workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learning-path-breadcrumb/ddm-template.ftl

```diff
@@ -0,0 +1,55 @@
+<script type="module">
+import "lms";
+document.addEventListener("DOMContentLoaded", function() {
+const title = document.querySelector(".title");
+var h1Text = document.getElementsByTagName('title')[0].textContent;
+title.innerHTML = h1Text;
+};
+</script>
+
+<#if (ObjectEntry_objectEntryId.getData())??>
```

Comment:
Add indentation and line break. 

In line 15, `learningPathName`  uses the previous variable 'learningPathId' on the url path, so we need to add a line break in there.